src/app/components/home/clip/clip.component.html

```diff
@@ -133,13 +141,14 @@
     [ngStyle]="{ 'height': imgHeight + 'px' }"
     (click)="videoClick.emit($event)"
     class="screencap"
+    [volume]="forceMute ? 0 : 1"
     onmouseover="this.play().catch((err) => {this.load()})"
     onmouseout="this.pause()"
     loop
   ></video>
 
   <video
-    *ngIf="noError && autoplay"
+    *ngIf="noError && autoplay && appInFocus"
```

Comment:
Will this cause the videos to reload again from disk?

Reply:
Oh! Good catch! I'm unsure 😅 

Even if that happens, it's probably better than running at 100% CPU to render everything in the background 😕 ... the _autoplay everything_ is a really CPU-hungry feature 😈